x/posts/keeper/msg_server_test.go

```diff
@@ -1491,14 +1491,6 @@ func (suite *KeeperTestSuite) TestMsgServer_RemovePostAttachment() {
 					"cosmos13t6y2nnugtshwuy0zkrq287a95lyy8vzleaxmd",
 					subspacestypes.NewPermission(types.PermissionModerateContent),
 				).Return(false)
-
-				suite.sk.EXPECT().HasPermission(
```

Comment:
It does not check the `EditOwnContent` permission when post owner does not match the message sender.